toolset/run-tests.py

```diff
@@ -182,6 +182,10 @@ def main(argv=None):
         nargs='+',
         default=[1, 10, 20, 50, 100],
         help='List of cached query levels to benchmark')
+    parser.add_argument(
+        '--benchmark-env',
+        default='none',
```

Comment:
Might need a better default here.

Reply:
Couldn't think of one and not sure it matters? But any suggestions?